app/src/main/java/com/keylesspalace/tusky/adapter/MentionTagAutoCompleteAdapter.java

```diff
@@ -44,6 +45,7 @@
         implements Filterable {
     private static final int ACCOUNT_VIEW_TYPE = 0;
     private static final int HASHTAG_VIEW_TYPE = 1;
+    private static final int EMOJI_VIEW_TYPE = 2;
```

Comment:
Maybe it makes sense to convert this to a Real Enum™ at this point?

Reply:
Sure, I'll do that real quick

Reply:
Oh, `BaseAdapter` mandates that a lot of the methods that deal with these constants use ints. It probably wouldn't be worthwhile to use an enum, then.